jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/DebugService.java

```diff
@@ -60,7 +60,7 @@
     private final PortForwardService portForwardService;
     private final ApplyService applyService;
     private String debugSuspendValue;
-    private String remoteDebugPort = DebugConstants.ENV_VAR_JAVA_DEBUG_PORT_DEFAULT;
+    private String containerDebugPort = DebugConstants.ENV_VAR_JAVA_DEBUG_PORT_DEFAULT;
```

Comment:
This name is still a bit confusing, a slight change might improve the semantics `debugPortInContainer`.